infinispan-client/infinispan-docker-compose.yaml

```diff
@@ -0,0 +1,13 @@
+version: '2'
+
+services:
+  infinispan-server:
+    image: jboss/infinispan-server:10.0.0.Beta3
+    ports:
+    - "11222:11222"
+    container_name: infinispan
+    networks:
+      - mynetwork
+networks:
```

Comment:
Please move `networks` before `services` as in client-docker-compose.yaml so both files have same format